subprojects/groovy-templates/src/main/groovy/groovy/text/GStringTemplateEngine.java

```diff
@@ -199,6 +199,7 @@ public Object run() {
             try {
                 groovyClass = loader.parseClass(new GroovyCodeSource(templateExpressions.toString(), "GStringTemplateScript" + counter.incrementAndGet() + ".groovy", "x"));
             } catch (Exception e) {
+                System.err.println("templateExpressions = " + templateExpressions);
```

Comment:
I guess this line should be reverted?

Reply:
Thanks for spotting that, yes I'll revert.